reference/7.0/Microsoft.PowerShell.Core/Start-Job.md

```diff
@@ -540,8 +540,30 @@ Accept wildcard characters: False
 
 ### -PSVersion
 
-Specifies a version. `Start-Job` runs the job with the version of PowerShell. The acceptable values
-for this parameter are: `2.0` and `3.0`.
+Specifies a version of PowerShell to use for running the job.
+Curently this parameter does not have any effect (job is run in PowerShell Core)
+except for value `5.1` which on Windows OS runs the job in a Windows PowerShell v5.1
+
+```powershell
```

Comment:
Examples are not allowed here by PlatyPS. Create an example in the example section of the article.

Reply:
moved to a new `Example 11`.